server/kepler_model_trainer.py

```diff
@@ -6,9 +6,10 @@
 import shutil
 from keras import backend as K
 from keras.callbacks import History
+import requests
 
 # Dict to map model type with filepath
-model_type_to_filepath = {"core_model": "/models/core_model", "dram_model": "/models/dram_model"}
+model_type_to_filepath = {"core_model": "models/core_model", "dram_model": "models/dram_model"}
```

Comment:
use absolute paths, e.g. `/var/lib/kepler-model-server`, this should also match the mount path [here](https://github.com/sustainable-computing-io/kepler-model-server/blob/main/manifests/deployment.yaml#L71)

Reply:
I can do that, but I am also using os.path.dirname(__file__) and appending it to /models/core_model which should give the absolute path of the file it currently is in no matter where. Does that not work?

Reply:
That should be fine, the path has to be deterministic so the deployment yaml and the source file can match.